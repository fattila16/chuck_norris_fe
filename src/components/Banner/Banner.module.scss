@import 'src/assets/scss/color-palette';
@import 'src/assets/scss/vars';

.container {
    display: flex;
    flex-direction: column;
    justify-content: center;
    align-items: center;
    object-fit: cover;
    background-color: map-get($active-theme , silk );
    margin-top: $navbar-height;
    padding-top: 20px;
    text-decoration: none;
    color: black;
    img{
        width: 15vw;
    }
}