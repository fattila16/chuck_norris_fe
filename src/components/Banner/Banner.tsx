import React from 'react';
import css from './Banner.module.scss';

const Banner: React.FC = () => {
    return (
        <a href="/" className={css.container}>
            <img src="/images/chuck_logo.png" alt="chuck_banner" />
            <h1>Chuck Norris Random Facts</h1>
        </a>
    );
};

export default Banner;
