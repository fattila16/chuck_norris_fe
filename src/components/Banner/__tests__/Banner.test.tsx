import React from 'react';
import { shallow } from 'enzyme';
import Banner from '../Banner';

describe.only('[Banner]', () => {
    it('Should render Banner component without error', () => {
        const component = shallow(<Banner />);
        expect(component.getElements()).toMatchSnapshot();
    });
});
