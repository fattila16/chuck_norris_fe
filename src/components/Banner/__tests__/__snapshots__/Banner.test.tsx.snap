// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Banner] Should render Banner component without error 1`] = `
Array [
  <a
    className="container"
    href="/"
  >
    <img
      alt="chuck_banner"
      src="/images/chuck_logo.png"
    />
    <h1>
      Chuck Norris Random Facts
    </h1>
  </a>,
]
`;
